common/OpTestUtil.py

```diff
@@ -2094,6 +2095,76 @@ def distro_name(self):
         else:
             return "unknown"
 
+    def get_distro_src(self, package ,dest_path, build_option=None):
+
+        host = self.conf.host()
+        if self.distro_name() == 'rhel':
+            path = tempfile.mkdtemp(prefix="optest_src")
```

Comment:
so this source path directory gets created inside host ?

Reply:
no  this is executing in host only just assign variable name